src/Approximations/decompositions.jl

```diff
@@ -220,7 +220,7 @@ julia> typeof(res[1]), typeof(res[2])
 """
 function decompose(S::LazySet{N},
                    partition::AbstractVector{<:AbstractVector{Int}},
-                   block_options
+                   block_options::AbstractVector
```

Comment:
This change is unrelated but I found it safer to restrict the type because we iterate over this object.